test/missing_semicolon.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
+    <script>
+        var xmlHttp = new XMLHttpRequest();
+        xmlHttp.open( "GET", '../src/js.cookie.js', false );
+        xmlHttp.send( null );
+        window.scriptContent = '(function (){ return {}; })() ';
+        window.scriptContent += xmlHttp.responseText;
+        window.scriptContent += ';window.ok = true;';
```

Comment:
I found a good resource in how to get errors from iframe in case you are interested: http://stackoverflow.com/a/14041297